### PR TITLE
Toggle group guidelines

### LINKED ISSUE #2028: Create design guidelines for toggle group
Background info: https://github.com/patternfly/patternfly/issues/2625
This one is probably going to be short.

### PR DESCRIPTION
**Pull request by gdoyle1**
Adding toggle group guidelines

Fixes #2028

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md [hunk 1]
```diff
@@ -3,5 +3,62 @@ id: Toggle group
...
+
+<img src="./img/elements-items.png" alt="Toggle group elements" width="1536"/>
+
+## Toggle group types
+
+1. Single select toggle group: Use to toggle between multiple items
+2. Multi select toggle group: Use to select multiple attributes
+3. Toggle items: Use to select a single action
+
+## Single select toggle group
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md:19]: Would me this an H3 underneath the H2 - "Toggle group types"
And I know you'll probably be say that then Usage and Examples won't be in the table of contents but I feel like that's fine! I think it makes more sense to have the group types be subheaders under the bigger "toggle group types" header

File: packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md [hunk 2]
```diff
@@ -3,5 +3,62 @@ id: Toggle group
...
+A **toggle group** is a group of controls that can be used to quickly switch between actions or states. 
+
+## Elements
+Toggle items can consist of icons, text, or both. Styling of toggles is dependent on the background color. 
+
+<img src="./img/elements-items.png" alt="Toggle group elements" width="1536"/>
+
+## Toggle group types
+
+1. Single select toggle group: Use to toggle between multiple items
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md:15]: Would add a link to each type, that would scroll the user down to where that content is on the page!

File: packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md (hunk 1 above)
Comment (gdoyle1) [reply packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md]: fixed!

File: packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md (hunk 2 above)
Comment (gdoyle1) [reply packages/v4/src/content/design-guidelines/components/toggle-group/toggle-group.md]: fixed!

**Review by mmenestr — APPROVED** [review]

**Pull request merged**